crates/ecs/src/lib.rs

```diff
@@ -1193,9 +1207,11 @@ impl<T: Debug + Send + Sync + 'static> ComponentVec for ComponentVecImpl<T> {
 #[derive(Default, Debug, Ord, PartialOrd, Eq, PartialEq, Copy, Clone, Hash)]
 pub struct Entity {
     id: usize,
-    _generation: usize,
+    //_generation: usize,
 }
 
+impl IsEnabled for Entity {}
```

Comment:
The recommended way to implement `IsEnabled` is to manually implement `Hash`, to ensure the invariant of 
> **One of the Hasher::write_{u8,u16,u32,u64,usize,i8,i16,i32,i64,isize} methods is invoked exactly once.**

is always upheld.

Could you replace our `#[derive(Hash)]` for `Entity` with a manual `Hash`-implementation, like the one shown in [the documentation](https://docs.rs/nohash-hasher/latest/nohash_hasher/trait.IsEnabled.html)?